src/git/parsers/blameParser.ts

```diff
@@ -28,7 +28,8 @@ export class GitBlameParser {
         data: string,
         repoPath: string | undefined,
         fileName: string,
-        currentUser: string | undefined
+        currentUser: string | undefined,
```

Comment:
Rather than separating `currentUser` and `currentEmail`, let's combine them into `currentUser: { name?: string; email?: string } | undefined`

Reply:
This applies to the other parsers as well